pyproject.toml

```diff
@@ -30,6 +30,7 @@ pyyaml = { version = ">=6.0", optional = true }
 requests = "*"
 pydantic = { extras = ["email"], version = ">=1.7,<3.0" }
 cappa = {version = ">=0.14.3,<0.20.0", optional = true}
+varname = "^0.13.1"
```

Comment:
@elliotgunton this should be an optional dependency

Reply:
I would agree if it's just while the new decorators are "experimental"? After that, I think we want it to be the golden path so should have minimal friction to use it (and so come packaged in)

Reply:
Any preference on what the `extras` group should be called in `pyproject.toml`? Going with `experimental` for now which will encompass current/future optional dependencies for any experimental features (aside from the cli which specifically pulls in cappa)

```
varname = {version = "^0.13.1", optional = true}

[tool.poetry.extras]
yaml = ["PyYAML"]
cli = ["cappa", "PyYAML"]
experimental = ["varname"]
```